testing/rundeck/APKBUILD

```diff
@@ -0,0 +1,86 @@
+# Contributor: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+# Maintainer: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+pkgname=rundeck
+pkgver=2.6.9
+pkgrel=0
+pkgdesc="Job scheduler and runbook automation. Enable self-service access to existing scripts and tools."
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+url="http://rundeck.org"
+arch="noarch"
+license="ASL 2.0"
+depends="openjdk8 bash ncurses5-libs"
+depends_dev=""
+makedepends="$depends_dev"
+install="$pkgname.pre-install"
+subpackages=""
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/rundeck/${pkgname}/archive/v${pkgver}.tar.gz
+	rundeck.initd
+	rundeck.confd
+	"
+builddir="$srcdir"/$pkgname-$pkgver
+JVERS=$pkgver-SNAPSHOT
+
+build() {
+	cd "$builddir"
+	export JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk
+	export TERM=dumb
+	bash -c "$builddir/gradlew build"
+}
+
+package() {
+	cd "$builddir"
+	install -m755 -D "$srcdir"/$pkgname.initd \
+		"$pkgdir"/etc/init.d/$pkgname || return 1
+	install -m644 -D "$srcdir"/$pkgname.confd \
+		"$pkgdir"/etc/conf.d/$pkgname || return 1
+	install -d -o $pkgusers -g $pkggroups \
+		"$pkgdir"/usr/bin \
+		"$pkgdir"/var/lib/rundeck/.ssh \
+		"$pkgdir"/var/lib/rundeck/exp/webapp \
+		"$pkgdir"/var/lib/rundeck/data \
+		"$pkgdir"/var/lib/rundeck/work \
+		"$pkgdir"/var/lib/rundeck/logs \
```

Comment:
> This is a place where rundeck store its jobs logs…

Then it’s okay.

> …wondering if I should use $pkgname instead of hardcoded "rundeck"

This is not needed, especially if changing these paths would also require modifying non-parametrized configs.